doc/source/installation.rst

```diff
@@ -14,9 +14,9 @@ Requirements
 Depending on the driver chosen, you may need to install additional OS packages.
 See ``INSTALL.rst``, which is created when initializing a new scenario.
 
-* :std:doc:`Ansible <ansible:index>` >= 2.4
+* :std:doc:`Ansible <ansible:index>` >= 2.5
 * Python 2.7
```

Comment:
@decentral1se it looks confusing: in which cases Python 2.7 would work that Python 3.6 wouldn't? It looks like this could be collapsed to just one line saying smth like `Python ==2.7 or >= 3.6` and `Ansible >=2.5`.

Reply:
Yep, right. Submitted https://github.com/ansible/molecule/pull/1820.